dp189/pages/category_page.py

```diff
@@ -0,0 +1,25 @@
+"""Category page."""
+
+from selenium.webdriver import Remote
+from dp189.pages.base_page import BasePage
+from dp189.components import LeftCategoryMenuComponent, SortByDropdownComponent, ShowNumberProductsDropdownComponent, \
+    ListViewComponent, GridViewComponent, CategoryProductWidgetComponent, ProductCompareLinkComponent
+
+
+class CategoryPage(BasePage):
+    """Category page class."""
+
+    def __init__(self, driver: Remote) -> None:
+        """All objects on product page.
+
+        :param driver: Remote
+        """
+        super().__init__(driver)
+        self.category_title = driver.find_element_by_id("content")
```

Comment:
I think that static locators must be in Locators.py module.

And, according to team convention, we should find elements in this way:
self.category_title = driver.find_element(By.ID, "content")

Reply:
Maybe it must be like this:
self.category_title = driver.find_element(*LocatorsCategoryPage.CATEGORY_TITLE)

in locators.py:
class LocatorsCategoryPage:
    CATEGORY_TITLE = (By.ID, "content")